src/stylesheets/screens/question.scss

```diff
@@ -0,0 +1,69 @@
+.questions {
+  width: 100vw;
+  height: 100vh;
+
+  &__header {
+    @include header-container();
+    height: rem(60px);
+
+    padding: rem(16px 16px 0);
+  }
+
+  &__header-right {
+    order: 2;
+    margin-left: auto;
+  }
+
+  &__container {
+    @include day-details-container();
```

Comment:
@rafayetn Please go over Noot why `@include` should be avoided as much as possible as we had this conversation recently. Your support is appreciated in this code review as well.

Reply:
@noor-ahmed `@extend` has a huge downside and it can make really long selectors after it compiles. More details can be found [here](https://webinista.com/updates/dont-use-extend-sass/). But there might be some cases when there is no option but to use `@extend`. We can use it then but we have to try to keep the selector as short as possible as long selectors have significant performance issues.

Reply:
@rafayetn thanks 👍